### ISSUE TITLE
Allow `convert_pt2e` to work with newer torchao ops

### ISSUE THREAD

**Issue by u-simha** [pytorch/ao#3058]
Hello! 

I wanted to raise an issue which I noticed when calling `convert_pt2e` on a model that was quantized using the [`AffineQuantizedMinMaxObserver`](https://github.com/pytorch/ao/blob/main/torchao/quantization/pt2e/_affine_quantization.py#L627) which uses the `quantize_affine` and `dequantize_affine` ops as opposed to the older `torch.fake_quantize_per_tensor` / `torch.fake_quantize_per_channel` ops. The newer ops allow for multiple granularities (per-block, per-token etc) which is beneficial but some of the existing utils and APIs don't work with the newer ops. 

One of the places I noticed was in the folding of batch norm into conv done [here](https://github.com/pytorch/ao/blob/main/torchao/quantization/pt2e/quantize_pt2e.py#L315), where the [`_append_qdq`](https://github.com/pytorch/ao/blob/main/torchao/quantization/pt2e/qat_utils.py#L172) method is called which searches for the older ops: 
https://github.com/pytorch/ao/blob/main/torchao/quantization/pt2e/qat_utils.py#L191

so if I have a model that uses the `AffineQuantizedMinMaxObserver`, I still see the batch norm ops after doing the `convert_pt2e` call:

```
...
call_function  quantize_affine_53                  torchao.quantize_affine             (conv2d_38, (1, 512, 7, 7), _scale53, _zero_point53, torch.uint8, 0, 255)                                                                                          {}
call_function  dequantize_affine_53                torchao.dequantize_affine           (quantize_affine_53, (1, 512, 7, 7), _scale53, _zero_point53, torch.uint8, 0, 255)                                                                                 {'output_dtype': torch.float32}
call_function  reshape_37                          aten.reshape.default                (div_36, [1, -1, 1, 1])                                                                                                                                            {}
call_function  div_37                              aten.div.Tensor                     (dequantize_affine_53, reshape_37)                                                                                                                                 {}
call_function  batch_norm_38                       aten.batch_norm.default             (div_37, layer4_1_bn1_weight, layer4_1_bn1_bias, layer4_1_bn1_running_mean, layer4_1_bn1_running_var, True, 0.1, 1e-05, True)                                      {}
call_function  relu__15                            aten.relu_.default                  (batch_norm_38,)     
...
```

This is the case in few other places:
https://github.com/pytorch/ao/blob/main/torchao/quantization/pt2e/quantizer/duplicate_dq_pass.py#L25

Is it possible to to also support the `quantize_affine` and `dequantize_affine` ops in these APIs?

**Comment by jcaip** [comment]
cc @jerryzh168 can you take a look at this please?

**Comment by jerryzh168** [comment]
sorry forgot to reply, yes I think we can add these new ops there. cc @andrewor14 can you add new ops to QAT flow?

for the duplicate_dq_pass, it's added by @kimishpatel I think, but I can help add that when I got a chance.
